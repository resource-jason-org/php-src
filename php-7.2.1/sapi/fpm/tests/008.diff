001+ sh: 1: -F: not found
002+ 
003+ Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
004+ Dynamic error
005+ 
006+ Warning: fsockopen(): unable to connect to 127.0.0.1:9009 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
007+ OnDemand error
008+ 
001- [%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
002- [%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
003- int(%d)
004- Dynamic ok
005- int(%d)
006- OnDemand ok
007- int(%d)
008- Static ok
009+ Warning: fsockopen(): unable to connect to 127.0.0.1:9010 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
010+ Static error