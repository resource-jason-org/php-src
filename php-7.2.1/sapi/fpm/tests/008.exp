[%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
[%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
int(%d)
Dynamic ok
int(%d)
OnDemand ok
int(%d)
Static ok