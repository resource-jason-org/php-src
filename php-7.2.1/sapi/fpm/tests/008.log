
---- EXPECTED OUTPUT
[%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
[%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
int(%d)
Dynamic ok
int(%d)
OnDemand ok
int(%d)
Static ok
---- ACTUAL OUTPUT
sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Dynamic error

Warning: fsockopen(): unable to connect to 127.0.0.1:9009 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
OnDemand error

Warning: fsockopen(): unable to connect to 127.0.0.1:9010 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Static error
---- FAILED
