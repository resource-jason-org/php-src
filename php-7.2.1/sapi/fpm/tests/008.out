sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Dynamic error

Warning: fsockopen(): unable to connect to 127.0.0.1:9009 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
OnDemand error

Warning: fsockopen(): unable to connect to 127.0.0.1:9010 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Static error